docs/getting_started/8_trigger_scripts/index.md

```diff
@@ -135,7 +135,7 @@ More details on our pages dedicated to [CLI](../../advanced/3_cli/index.md).
 
 In Windmill, webhooks are autogenerated for each Script and Flow, providing either asynchronous or synchronous execution modes. 
 
-These webhooks accept incoming HTTP requests, allowing users to easily trigger their Windmill workflows from external services by simply sending a POST request to the appropriate webhook URL. Their purpose is to have the script run when it receives an input from its associated webhook.
+These webhooks accept incoming HTTP requests, allowing users to easily trigger their Windmill workflows from external services by simply sending a POST request to the appropriate webhook URL and token. Their purpose is to have the script run when it receives an input from its associated webhook.
```

Comment:
authentified webhook URL (requires passing a token as Bearer or query arg, see more at [link to ref])